src/layers/generic.h

```diff
@@ -288,8 +288,11 @@ class Output : public LayerBase, public IUnaryLogitLayer, public IHasShortList {
 // EncoderDecoderLayerBase, which knows to pass on all required parameters from options.
 class Embedding : public LayerBase, public IEmbeddingLayer {
   Expr E_;
+  Expr FE_;
```

Comment:
Let's try to make variable names less obtuse over time.

Reply:
Changed the name to something more readable